config/mypy.ini

```diff
@@ -1,6 +1,6 @@
 [mypy]
 ignore_missing_imports = true
-exclude = tests/fixtures/
+exclude = tests/fixtures/|tests/test_git/_repo
```

Comment:
The `fixtures` directory is exactly made for this! Can you move `_repo` inside it?
You have some `Path` instances already defined in `tests/__init__.py` that you can import:

```python
from tests import FIXTURES_DIR
```

Reply:
sure. there is no `test/fixtures` folder.  but I can create one